### PR TITLE
init sidebar

### PR DESCRIPTION
**Pull request by Madhurapabu**

### CODE REVIEW

**Comment by chxru** [comment]
@Madhurapabu fix the conflicts will you?

**Comment by Madhurapabu** [comment]
> 
> 
> @Madhurapabu fix the conflicts will you?

@CharukaHS It would be fine if you had your contribution

File: web-app/components/sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,247 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```
Comment (chxru) [inline web-app/components/sidebar.tsx]: /* eslint-disable @typescript-eslint/no-unused-vars */ remove the tag, only use single line lint disable code if it is necessary

File: web-app/components/sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  Heading,
+  Icon,
+  IconButton,
+  useDisclosure,
+  Input,
+  InputGroup,
+  InputLeftElement,
+} from "@chakra-ui/react";
+import React, { useContext } from "react";
+import { HamburgerIcon } from "@chakra-ui/icons";
```
Comment (chxru) [inline web-app/components/sidebar.tsx]: react-icons is already available, why bothering with another package for icons?

File: web-app/components/sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+import { useRouter } from "next/dist/client/router";
```
Comment (chxru) [inline web-app/components/sidebar.tsx]: use `import { useRouter } from 'next/router' `

File: web-app/components/sidebar.tsx [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+import { IconType } from "react-icons/lib";
+import { FaBell } from "react-icons/fa";
+import { FiSearch } from "react-icons/fi";
```
Comment (chxru) [inline web-app/components/sidebar.tsx]: Use same type of icons for consistent design pattern

File: web-app/components/sidebar.tsx [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+          {/* Search  */}
+          <InputGroup
+            w="40"
+            size="sm"
+            display={{ base: "none", md: "flex" }}
+            mr={8}
+            borderWidth="0px"
+          >
+            <InputLeftElement>
+              {" "}
```
Comment (chxru) [inline web-app/components/sidebar.tsx]: unnecessary  `{" "} `

**Review by chxru — CHANGES_REQUESTED** [review]
I've mentioned few points that should be optimized, please check fix them

**Pull request merged**